docs/docs/modules/chains/sql_db.md

```diff
@@ -0,0 +1,31 @@
+# SQLDatabaseChain
+
+The `SqlDatabaseChain` allows you to answer questions over a SQL database.
+This example uses Chinook database, which is a sample database available for SQL Server, Oracle, MySQL, etc.
+To set it up follow the instructions on https://database.guide/2-sample-databases-sqlite/.
+
+```typescript
+import { DataSource } from "typeorm";
+import { OpenAI } from "langchain/llms";
+import { SqlDatabase } from "langchain/sql_db";
+import { SqlDatabaseChain } from "langchain/chains";
+
+export const run = async () => {
+  const datasource = new DataSource({
+    type: "sqlite",
+    database: "Chinook.db",
+  });
+
+  const db = await SqlDatabase.fromDataSourceParams({
+    appDataSource: datasource,
+  });
+
+  const chain = new SqlDatabaseChain({
+    llm: new OpenAI({ temperature: 0 }),
+    database: db,
+  });
+
+  const res = await chain.run("How many tracks are there?");
+  console.log(res);
```

Comment:
Let's also add the db destroy to any examples @agola11

Reply:
will do! And will also add to tests